spec/plugins_spec.rb

```diff
@@ -8,9 +8,9 @@
   title description author git repository
 }
 
-describe("plugin manifests") do
-  Dir["_plugins/*"].each do |plugin|
-    plugin_info = SafeYAML.load_file(plugin)
+Dir["_plugins/*"].each do |plugin|
+  plugin_info = SafeYAML.load_file(plugin)
+  describe(plugin) do
```

Comment:
Flipped this around so that `it` refers to the specific plugin manifest filename, rather than the string `"plugin manifests"`